wordproof-timestamp/includes/Page/SettingsPage.php

```diff
@@ -37,18 +30,34 @@ public function generateSettingsPage() {
         wp_localize_script('wordproof.admin.js', 'wordproofSettings', [
             'network' => get_option('wordproof_network', false),
             'certificateText' => CertificateHelper::getCertificateText(),
-            'saveChanges' => __('Save Changes')
+            'saveChanges' => __('Save Changes', 'wordproof-timestamp')
```

Comment:
I see `'wordproof-timestamp'` being used quite often. Maybe make a constant out of this or wrap the function `__`?